src/ksync/ksync_sock.cc

```diff
@@ -599,19 +682,28 @@ size_t KSyncSockNetlink::SendTo(KSyncBufferList *iovec, uint32_t seq_no) {
                              nl_client_->cl_buf_offset));
     UpdateNetlink(nl_client_, bulk_buf_size_, seq_no);
 
-    //WINDOWSFIX boost::asio::netlink::raw::endpoint ep;
-	return 0;//WINDOWSFIX sock_.send_to(*iovec, ep);
+#ifndef _WINDOWS
+    boost::asio::netlink::raw::endpoint ep;
+    return sock_.send_to(*iovec, ep);
+#else
+    boost::system::error_code ec;
+    size_t sz = boost::asio::write(pipe_, *iovec, ec);
+    if (ec) {
```

Comment:
Does `send_to` throws an exception on failure? If it does, does agent code expects this exception to be thrown somewhere?